stats/DataAggregation.md

```diff
@@ -0,0 +1,71 @@
+# Data Aggregation API Overview
+The stats library allows users to define views to configure how data are aggregated and broken down.
+The core data types used are:
+* `Aggregation`: defines the aggregation type (e.g. `Distribution`, `Sum`, `Mean`).
+* `View`: defines how individual measurements are broken down by tags (similar to labels in some 
+other systems) and aggregated.
+
+## Aggregation
+An Aggregation describes how data collected is aggregated.
+
+The library SHOULD provide support for multiple types of Aggregations:
+* `Count`: counts the number of measurements recorded.
+* `Sum`: indicates that data collected and aggregated with this `Aggregation` will be summed up.
+* `Mean`: indicates that data collected and aggregated with this `Aggregation` will calculate the
+mean value.
+* `Max`: indicates that data collected and aggregated with this `Aggregation` will calculate the 
+maximum value recorded.
+* `LastValue`: indicates that data collected and aggregated with this `Aggregation` will 
```

Comment:
fyi @jkschneider there are a couple new things here as well an explanation of views if you have some time to comment.

Reply:
Thanks Adrian.